.github/CODEOWNERS

```diff
@@ -19,10 +19,53 @@ subprojects/enterprise-logging/     @gradle/ge-build-insights @gradle/ge-testing
 subprojects/enterprise-workers/     @gradle/ge-build-insights
 
-# Testing
-subprojects/testing-junit-platform/ @gradle/ge-testing
-subprojects/testing-jvm/            @gradle/ge-testing
+# JVM platform
+subprojects/code-quality/           @gradle/bt-jvm
+subprojects/ear/                    @gradle/bt-jvm
+subprojects/jacoco/                 @gradle/bt-jvm
+subprojects/java-compiler-plugin/   @gradle/bt-jvm
+subprojects/jvm-services/           @gradle/bt-jvm
+subprojects/language-groovy/        @gradle/bt-jvm
+subprojects/language-java/          @gradle/bt-jvm
+subprojects/normalization-java/     @gradle/bt-jvm
+subprojects/platform-jvm/           @gradle/bt-jvm
+subprojects/plugins/                @gradle/bt-jvm
+subprojects/scala/                  @gradle/bt-jvm
+subprojects/testing-junit-platform/ @gradle/bt-jvm @gradle/ge-testing
+subprojects/testing-jvm/            @gradle/bt-jvm @gradle/ge-testing
 
-# Release notes
+# Software platform
+subprojects/build-init/             @gradle/bt-jvm
+subprojects/ivy/                    @gradle/bt-jvm
+subprojects/maven/                  @gradle/bt-jvm
+subprojects/platform-base/          @gradle/bt-jvm
+subprojects/publish/                @gradle/bt-jvm
+subprojects/reporting/              @gradle/bt-jvm
+subprojects/resources/              @gradle/bt-jvm
+subprojects/resources-gcs/          @gradle/bt-jvm
+subprojects/resources-s3/           @gradle/bt-jvm
+subprojects/resources-sftp/         @gradle/bt-jvm
+subprojects/security/               @gradle/bt-jvm
+subprojects/signing/                @gradle/bt-jvm
+subprojects/testing-base/           @gradle/bt-jvm
+subprojects/version-control/        @gradle/bt-jvm
+
+# Documentation
 subprojects/docs/src/docs/release/notes.md @hythloda
 subprojects/docs/                          @nathan-contino
+
+subprojects/docs/src/docs/userguide/core-plugins/base_plugin.adoc                 @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/build_dashboard_plugin.adoc      @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/build_init_plugin.adoc           @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/checkstyle_plugin.adoc           @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/codenarc_plugin.adoc             @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/distribution_plugin.adoc         @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/ear_plugin.adoc                  @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/jacoco_plugin.adoc               @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/java_gradle_plugin.adoc          @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/pmd_plugin.adoc                  @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/core-plugins/war_plugin.adoc                  @gradle/bt-jvm
+subprojects/docs/src/docs/userguide/dep-man/                                      @gradle/bt-jvm
```

Comment:
I see we're assigning dependency management to our team, as well as assigning the software platform to us as well. In that case, does it make sense to also add `subprojects/dependency-management` to the `#Software Platform` list above? 

Also, we should consider splitting up the docs sections by platform as well.

Reply:
> In that case, does it make sense to also add subprojects/dependency-management to the #Software Platform list above?

Maybe, I was trying to treat all of docs as a single thing and just plucking out pieces.